frontend/src/main/web/lib/components/glossary/DataTable.jsx

```diff
@@ -201,20 +202,29 @@ var DataTable = React.createClass({
       var entry = this._getGlossaryEntry(id);
       var value = _.get(entry, field.field);
       if (readOnly) {
-        return <span className="mh1/2" key={key}>{value}</span>;
-      } else {
-        return (
-          <InputCell
-            value={value}
-            id={id}
-            key={key}
-            maxLength={maxLength}
-            placeholder={placeholder}
-            rowIndex={rowIndex}
-            field={field.field}
-            onFocusCallback={this._onRowClick}/>
-        );
+        var span = <span className="mh1/2" key={key}>{value}</span>;
```

Comment:
If meant for parent scope, declare it at top. If meant for just this block, use `let`